tests/test_get_next_results.py

```diff
@@ -13,12 +13,15 @@ def test_get_next_results(base_url, selenium):
     assert len(page.result_sets) == 10
 
     page.get_next_ten_results()
+    assert page.results_visible
```

Comment:
Rather than an explicit check for the results to be visible, I would suggest adding this as a wait inside `get_next_ten_results` or the page object it creates. The same comment applies to the remaining changes.